### DISCUSSION TITLE
on moddable 2, which is the Boot button

### DISCUSSION THREAD

**Discussion by bap293** [Moddable-OpenSource/moddable#690] [Getting started] [answered]
Getting this error:
![image](https://user-images.githubusercontent.com/89415466/130617716-1fee8f7a-923a-4224-84fd-20b42cb7b984.png)

Following instructions to get device into boot mode, tried both buttons (flash and reset) to no avail.  Which button should I be pressing to get into boot mode?

**Comment by mkellner** [comment, marked as answer]
`Flash` is the button. When the ....____.... is going, hold the flash button.

Also, make sure the UPLOAD_PORT is set to the proper device. In your truncated image above, it looks like you have it set for COM7.

**Reply by bap293** [reply]
Pressing the flash button did it  Thanks! Pressing that button when ...___... is much easier too.

**Reply by phoddie** [reply]
There's a great documentation for Moddable Two [here](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/devices/moddable-two.md) (in this repository). It includes photo of the board which shows the pin outs and labels for both the flash and reset buttons.

**Reply by bap293** [reply]
Thank you sir! (yea I'm old fashioned).
